CRM/Core/Payment/iATSService.php

```diff
@@ -427,13 +427,10 @@ protected function convertParams($params, $method) {
 
     foreach ($convert as $r => $p) {
       if (isset($params[$p])) {
-        $request[$r] = htmlspecialchars($params[$p]);
+        $request[$r] = str_replace(str_split(";"), '', $params[$p]);
       }
     }
-    // The "&" character is badly handled by the processor,
-    // so we sanitize it to "and"
-    $request['firstName'] = str_replace('&', 'and', $request['firstName']);
-    $request['lastName'] = str_replace('&', 'and', $request['lastName']);
+    
```

Comment:
I think we probably want to leave these in now, for the occasional person with a shared card that uses the ampersand.